test/application/rooms.js

```diff
@@ -0,0 +1,25 @@
+var assert = require('assert');
+var room;
+
+describe('Rooms features', function () {
+	describe('Rooms', function () {
+		describe('Rooms.get', function () {
+			it('should be a function', function () {
```

Comment:
We could have renamed the functions to have more formal names...
Actually, we can still redo it with appropriate names... Suggest some?